libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h

```diff
@@ -211,8 +211,6 @@ NS_SWIFT_NAME(SalesforceManager)
  */
 @property (nonatomic, copy, nullable) NSString *idpAppURIScheme NS_SWIFT_NAME(identityProviderURLScheme);
 
-@property (nonatomic, copy, nullable) NSString *idpKeychainGroup NS_SWIFT_NAME(identityProviderKeychainGroup);
```

Comment:
Originally added this so that the SP app could use it for user tracking, but we don’t need that anymore so we could pass the keychain group from SPConfig through in the URL